android/res/values/strings.xml

```diff
@@ -463,6 +463,10 @@
 	<string name="current_location_unknown_message">Current location is unknown. Maybe you are in a building or in a tunnel.</string>
 	<string name="current_location_unknown_continue_button">Continue</string>
 	<string name="current_location_unknown_stop_button">Stop</string>
+	<string name="stop_navigation_title">Stop navigation?</string>
```

Comment:
Please use `data/strings/strings.txt` file to add new strings.
Also, you can generate new translations by installing `trans` and then using this command:
`DELIM=" = " tools/unix/translate_categories.txt`
If translation API bans/rate limits your IP address, then you can temporarily change it to another one (or use VPN).

Reply:
Then you can run `tools/unix/generate_translations.sh` to regenerate strings.